lib/ffmpeg/vaapi/vpp.py

```diff
@@ -42,6 +42,14 @@ def gen_vpp_opts(self):
     elif self.vpp_op in ["overlay"]:
       vpfilter.append("[1:v]hwupload[v1];")
       vpfilter[-1] += "[0:v][v1]overlay_vaapi=alpha={a}".format(a = mapRange(self.alpha, [0.0, 255.0], [0.0, 1.0]))
+    elif self.vpp_op in ["pad"]:
+      vpfilter.append("pad_vaapi")
+      if hasattr(self, "padw") and hasattr(self, "padh"):
+        vpfilter[-1] += f"=width={self.padw}:height={self.padh}"
+        if hasattr(self, "x") and hasattr(self, "y"):
+          vpfilter[-1] += f":x={self.x}:y={self.y}"
+          if hasattr(self, "color"):
+            vpfilter[-1] += f":color={self.color}"
```

Comment:
What about `color` setting without `x` and `y`?

Reply:
Updated, thx